solr/webapp/web/css/angular/common.css

```diff
@@ -591,21 +591,6 @@ pre.syntax .tex .formula
   opacity: 0.5;
 }
 
-#content .tree li,
-#content .tree ins
-{
-  background-color: transparent;
-  background-image: url( ../../img/tree.png );
-  background-repeat: no-repeat;
-}
-
-#content .tree li
-{
-  background-position: -54px 0;
-  background-repeat: repeat-y;
-  line-height: 22px;
-}
-
```

Comment:
Included jstree CSS file which avoided needing this. I didn't remove `tree.png`. I think there are a BUNCH of img files that can be cleaned up, but want to do that separate.